app/Models/Listing.php

```diff
@@ -168,12 +168,22 @@ public function unlike(User $user)
     public function getLikesTextAttribute()
     {
         $likeCount = $this->likes->count();
+        $currentUser = auth()->user();
 
         if ($likeCount === 0) {
             return 'No one liked this post yet';
-        } elseif ($likeCount <= 2) {
-            $likedBy = $this->likes->pluck('user.name')->implode(', ');
-            return "$likedBy liked this post";
+        } elseif ($likeCount === 1 && $this->likes->contains('user_id', $currentUser->id)) {
```

Comment:
make an enum for $likeCount values

Reply:
also message use lang

Reply:
sorry the $likeCount condition should be check from enum
if ($likeCount === LikeCount::zero) {

Reply:
after fix. Please merge. Thank you